websub-native/src/main/java/io/ballerina/stdlib/websub/RequestHandler.java

```diff
@@ -37,41 +20,50 @@
 import static io.ballerina.runtime.api.utils.StringUtils.fromString;
 
 /**
- * {@code SubscriberNativeOperationHandler} handles the native method execution.
+ * {@code RequestHandler} is a wrapper object used for service method execution.
  */
-public class SubscriberNativeOperationHandler {
+public class RequestHandler {
```

Comment:
Can't we rename this the same name `HttpToWebsubAdaptor ` or maybe `NativeHttpToWebsubAdaptor `